### PR TITLE
initial commit, add plots, compare cell types differential_prio, upda…

### PR DESCRIPTION
**Pull request by antschum**
Added plots. 
Update plots, differential prioritisation and feature importances in tutorial.

### CODE REVIEW

File: augurpy/differential_prioritization.py [hunk 1]
```diff
@@ -33,17 +33,38 @@ def predict_differential_prioritization(
     Returns:
         Results object containing mean augur scores.
     """
+    # compare cell types available
```
Comment (Zethson) [inline augurpy/differential_prioritization.py]:
```suggestion
    # compare available cell types
```

File: augurpy/plotting/differential_prioritization.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from matplotlib import pyplot as plt
+
+
+def plot_differential_prioritization(results, top_n=None):
```
Comment (Zethson) [inline augurpy/plotting/differential_prioritization.py]: Usually we try to always make the matplotlib axes available as a parameter and a couple more (see e.g. what Scanpy's plots expose). This helps with custom plots where you e.g. want to create an image with several plots.

Would also return the axes here I guess.

File: augurpy/plotting/lollipop.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+from matplotlib import pyplot as plt
+
+
+def plot_lollipop(results: dict):
```
Comment (Zethson) [inline augurpy/plotting/lollipop.py]: same plot comments see above

File: augurpy/plotting/scatterplot.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+from matplotlib import pyplot as plt
+
+
+def plot_scatterplot(results1, results2, top_n=None):
```
Comment (Zethson) [inline augurpy/plotting/scatterplot.py]: see above

**Review by Zethson — CHANGES_REQUESTED** [review]
Generally we can remove the "plot_" prefix from the function names because they are already in the "plot/pl" module. 

Thanks!

File: augurpy/plotting/differential_prioritization.py (hunk 2 above)
Comment (antschum) [reply augurpy/plotting/differential_prioritization.py]: Ah, I'll look into that!

**Pull request merged**